examples/hacker_news_assets/README.md

```diff
@@ -1,109 +1,46 @@
-# Hacker News Demo Jobs - Software-Defined Assets Version
+# Hacker News Demo
 
-This repo used Dagster's experimental software-defined asset APIs to set up a "realistic" example of using dagster in a production world, integrating with
-many of its features, including:
+This is meant to be a full "realistic" demo of Dagster, that takes advantage of many of its features, including:
 
-- Schedules / Sensors
-- Asset Materializations
+- Software-defined assets
+- Schedules
```

Comment:
is there a schedule in the example? the schedules folder is empty

Reply:
there actually is - it's in core/__init__.py